### PR TITLE
heres my my question

### PR DESCRIPTION
**Pull request by garunanurag**
please let me know what can i do to improve

### CODE REVIEW

**Review by dakshkhetan — CHANGES_REQUESTED** [review]
@garunanurag You haven’t followed the contribution rules. Go through them in the documentation and also, look at other's format on how to add write your question in the `<ol>` list.

**Comment by garunanurag** [comment]
> @garunanurag You haven’t followed the contribution rules. Go through them in the documentation and also, look at other's format on how to add write your question in the `<ol>` list.

i  have to put question in ol list  and is  answer part in the right format?

**Comment by garunanurag** [comment]
i have  made some changes

File: javascript/index.html [hunk 1]
```diff
@@ -73,6 +73,7 @@ <h4>
                     to the local variable <code>self</code> remains in scope and is accessible there.
                 </p>
             </li>
+            <li> what is event loop in JavaScript</li>>
```
Comment (deepak-singh343) [inline javascript/index.html]: There is an unnecassry ">" at the end of the line please remove it

**Review by deepak-singh343 — CHANGES_REQUESTED** [review]

File: javascript/answers/Anurag-Pandey-q35.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+            <img src="../../assets/CN_logo.png" height='72px' alt="Coding Ninjas Logo">
+        </div>
+
+        <div>
+            <p>
+                <small>
+
+                    <a href="../../index.html">Home</a>
+                    &gt; <a href="../index.html">JavaScript Questions</a>
+                    &gt; What is NaN? What is its type? How can you reliably test if a value is equal to NaN?
```
Comment (dakshkhetan) [inline javascript/answers/Anurag-Pandey-q35.html:32]: Change this to your question title.

File: javascript/answers/Anurag-Pandey-q35.html [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        gtag('config', 'UA-157837769-1');
+        </script>
+        
+        <title>
+            JavaScript | InterQues | Coding Ninjas
+        </title>
+
+        <link rel="shortcut icon" type="image/png" href="../../assets/favicon.png"/>
+
+        <meta name="description" content="What is NaN? What is its type? How can you reliably test if a value is equal to NaN?">
```
Comment (dakshkhetan) [inline javascript/answers/Anurag-Pandey-q35.html:19]: Change content="" to your question title

**Review by dakshkhetan — CHANGES_REQUESTED** [review]
@garunanurag Why have you made changes in the GauravDwivedi-q9 file?

**Comment by deepak-singh343** [comment]
you have sent me by mistake this is not my PR

**Comment by dakshkhetan** [comment]
Closing this PR due to no changes done.
You can raise another after making requested changes.

**Pull request closed without merging**